### PR TITLE
Add round Avatar optional prop

### PR DESCRIPTION
**Pull request by antonio-garcia-sonarsource**
Sonarcloud is going to start using completely rounded avatars in a couple of places inside the New UI. In order to make this easy and clean I added an optional round prop to the GenericAvatar component.

**Checklist**

* [ ] Functional validation
* [ ] Documentation update
* [x] Update changelog

### CODE REVIEW

File: components/ui/GenericAvatar.tsx [hunk 1]
```diff
@@ -43,6 +44,7 @@ export default function GenericAvatar({ className, name, size }: Props) {
       className={classNames(className, 'rounded')}
       style={{
         backgroundColor: color,
+        borderRadius: round ? '50%' : undefined,
```
Comment (gregaubert) [inline components/ui/GenericAvatar.tsx:47]: does the `undefined` value override the `.rounded` class radius ?

Comment (gregaubert) [reply components/ui/GenericAvatar.tsx:47]: It would also be good to add a test for that new prop

**Review by gregaubert — CHANGES_REQUESTED** [review]
Looks good!
Just add the missing test, and confirm that the `undefined` value doesn't break the rounded class

File: components/ui/GenericAvatar.tsx (hunk 1 above)
Comment (antonio-garcia-sonarsource) [reply components/ui/GenericAvatar.tsx:47]: No problem regarding the `.rounded` class border-radius as the element border-radius is not even there when using undefined. I just tested it and there's no problem there.

Comment (gregaubert) [reply components/ui/GenericAvatar.tsx:47]: Nice :)

Comment (antonio-garcia-sonarsource) [reply components/ui/GenericAvatar.tsx:47]: I just added the missing test 👍

**Review by gregaubert — APPROVED** [review]

**Pull request merged**